models/model.js

```diff
@@ -69,13 +69,31 @@ const fetchCommentsByArticleId= (article_id) =>{
   if (!article_id ||!username ||!body) {
     return Promise.reject({ status: 400, msg: 'Incomplete comment' });
   } 
-   
+
+  else{   
     return db.query(`INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;`, [article_id, username, body])
     .then((result) => { 
      
         return result.rows[0]    
   })
 }
+  }
+
+const fetchVotedArticlesById = (article_id, votes) => {
+if (typeof votes !== 'number') {
```

Comment:
The passed votes being the wrong datatype would cause a PSQL error that could be caught in the PSQL handling middleware in your app.js file.